test/functional/test_framework/messages.py

```diff
@@ -127,10 +127,16 @@ def deser_vector(f, c):
     return r
 
 
-def ser_vector(l):
+# ser_function_name: Allow for an alternate serialization function on the
+# entries in the vector (we use this for serializing the vector of transactions
+# for a witness block).
```

Comment:
does this need adjusting?

Reply:
How about fixing it in https://github.com/dashpay/dash/pull/4181#discussion_r641980281?

Reply:
I'm fine with that.